src/js/ia/IAPage.js

```diff
@@ -360,6 +361,10 @@
                         }
                     });
 
+                    $("body").on('click', ".userpage-link a", function(evt) {
```

Comment:
Okay, this is something way bigger than we should get into here, I figured there was probably a reason. :)

A more modular lifecycle pattern that frameworks typically use is something like 1) create view 2) render elements 3) bind events to elements 4) remove events 5) remove elements 6) destroy view.